src/libraries/MathLib.sol

```diff
@@ -8,33 +8,33 @@ uint256 constant WAD = 1e18;
 /// @custom:contact security@morpho.org
 /// @notice Library to manage fixed-point arithmetic.
 library MathLib {
-    /// @dev (x * y) / WAD rounded down.
+    /// @dev Returns (`x` * `y`) / `WAD` rounded down.
```

Comment:
To me we should add `@param` in case there are many parameters or else that those paramaters require explanations.

Reply:
What do you mean by "many" ? More than 1 ? I'm not against your suggestion but I want us to arrive at a conclusion on this.

More specifically, what do you think about the natspecs comments of:
1. the constructor of the Morpho contract
2. the functions in the IIrm

Reply:
Good question, in my mind we can avoid using @param when the description of input parameters and what returns the functions can fit in one line. Else we should use the `@param` statement. 

But "when the description of input parameters and what returns the functions can fit in one line" does not generalize well. 

For the constructor and IIrm we could have a one liner.

Perhaps to avoid debates, we should always use `@param` whatever the situation.

Reply:
I like your rule, I think we should apply it to the IIrm and to the constructor

Reply:
Which one? 😄

This one?

> we can avoid using @param when the description of input parameters and what returns the functions can fit in one line. Else we should use the @param statement.

or this one?

> Perhaps to avoid debates, we should always use @param whatever the situation.

Reply:
Oops sorry, I meant the first one. Let's not put `@param` comments that are useless if we can avoid it